projects/ui/src/shim.cds-core.scss

```
@@ -499,68 +499,128 @@
-  --clr-label-info-font-color: var(--clr-label-font-color-light); // var(--clr-color-action-800);
-  --clr-label-info-border-color: var(--cds-alias-status-info); // var(--clr-color-action-800);
-
-  --clr-label-success-bg-color: var(--cds-alias-status-tint); // var(--clr-color-success-50);
-  --clr-label-success-font-color: var(--clr-label-font-color-light); // var(--clr-color-success-800);
-  --clr-label-success-border-color: var(--cds-alias-status-success); // var(--clr-color-success-800);
-
-  --clr-label-warning-bg-color: var(--cds-alias-status-tint); // var(--clr-color-warning-100);
-  --clr-label-warning-font-color: var(--clr-label-font-color-light); // var(--clr-color-neutral-900);
-  --clr-label-warning-border-color: var(--cds-alias-status-warning); // var(--clr-color-warning-800);
-
-  --clr-label-danger-bg-color: var(--cds-alias-status-tint); // var(--clr-color-danger-100);
-  --clr-label-danger-font-color: var(--clr-label-font-color-light); // var(--clr-color-danger-900);
-  --clr-label-danger-border-color: var(--cds-alias-status-danger); // var(--clr-color-danger-900);
+  --clr-label-padding: var(--cds-global-space-3) var(--cds-global-space-5);
+  --clr-label-gap-size: var(--cds-global-space-5);
+  --clr-label-box-shadow: 0;
+
+  // ATM - remove and use as default or think a better way express clicked state
+  --clr-label-pressed-transform: translateY(calc(1 * var(--cds-global-space-1)));
+
+  --clr-label-font-size: var(--clr-p6-font-size);
+  --clr-label-font-weight: var(--cds-global-typography-font-weight-regular);
+  --clr-label-letter-spacing: var(--clr-p6-letter-spacing);
+  --clr-label-line-height: var(--clr-p6-line-height);
+  --clr-label-font-color-light: var(--cds-alias-typography-color-500, var(--cds-global-typography-color-500));
+
+  --clr-label-clickable-font-size: var(--clr-label-font-size);
+  --clr-label-clickable-font-weight: var(--clr-label-font-weight);
+  --clr-label-clickable-letter-spacing: var(--clr-label-letter-spacing);
+  --clr-label-clickable-line-height: var(--clr-label-line-height);
+
+  --clr-label-default-border-color: var(--cds-alias-utility-gray);
+  --clr-label-default-bg-color: var(--cds-alias-object-opacity-0);
+
+  --clr-label-bg-hover-color: var(--cds-alias-object-interaction-background-hover);
+  --clr-label-hover-color: var(--cds-alias-object-interaction-color-hover);
+  --clr-label-bg-active-color: var(--cds-alias-object-interaction-background-active);
+  --clr-label-active-color: var(--cds-alias-object-interaction-color-active);
+  --clr-label-bg-disabled-color: var(--cds-alias-object-interaction-background-disabled);
+  --clr-label-disabled-color: var(--cds-alias-typography-disabled);
+  --clr-label-disabled-border-color: var(--cds-alias-utility-disabled, var(--cds-alias-object-interaction-color-disabled));
+
+  --clr-label-gray-bg-color: var(--cds-alias-utility-gray);
+  --clr-label-gray-color: var(--cds-alias-typography-color-500, var(--cds-global-typography-color-500));
```

Can you add `border-color` for all non status based labels?
That change have to be reflected in `_properties.labels.scss` as well.
To use `border-color` properly in `_labels.clarity.scss` at line 21 (`@include mixins.css-var(border-color, $cssVarBgColor, $sassVarBgColor, variables.$clr-use-custom-properties);`)

```suggestion
  --clr-label-gray-bg-color: var(--cds-alias-utility-gray);
  --clr-label-gray-bg-color: var(--clr-label-gray-bg-color);
  --clr-label-gray-color: var(--cds-alias-typography-color-500, var(--cds-global-typography-color-500));
```